pages/guide.md

```diff
@@ -106,13 +107,17 @@ Federally operated domains do not all end in `.gov`, `.mil`, or `.fed.us`. Some
 
 ### What about domains that are only used to redirect visitors to other websites?
 
-These domains must enable port 443, use and enforce HTTPS, and follow all the same requirements and guidelines as domains used to host websites and APIs, including HSTS and preloading.
+These domains must enable port 443 and use properly configured HTTPS.
 
-### Do domains that redirect to an external domain first need to redirect internally to the secure form of itself?
+They must follow all the same requirements and guidelines as domains used to host websites and APIs, including HSTS and preloading.
 
-It is not required, for example, to redirect from `http://example.gov:80` to `https://example.gov:443` before redirecting to `https://another-example.gov:443`. However, doing so enables the connecting client to see and cache the HSTS header on `example.gov`, which may not otherwise be seen. 
+### Do domains that redirect to other external domains need to redirect internally to HTTPS before redirecting externally?
 
-Redirecting internally is a [prerequisite to preloading a domain](https://hstspreload.org/#submission-requirements). As only second-level domains can be preloaded, this practice is recommended for second-level domains.
+Not generally, but it is practically required in order to preload a second-level domain.
+
+For example, it is not required by M-15-13 to redirect from `http://example.gov:80` to `https://example.gov:443` before redirecting to `https://another-example.gov:443`. However, doing so enables the connecting client to see and cache the HSTS header on `example.gov`, which it may not otherwise see.
+
+However, doing an internal redirect first **is required** [to automatically preload second-level domains](https://hstspreload.org/#submission-requirements), and so this practice is recommended for second-level domains.
```

Comment:
It is technically possible to preload non-second-level-domains (and I've done so before) but it requires manual interaction with the Chrome HSTS preload team. I adjusted it to be a little more precise about that.